### ISSUE TITLE
HiDPI anomalies 1.20

### ISSUE THREAD

**Issue by markd89** [mate-desktop/marco#436]
I just got a "new" laptop Dell XPS 15 9550 (2015 version) with a 4K display. I am running Debian 9/Mate 1.20.

If I use any of the Marco compositors, things look pretty good except for an annoying horizontal line through each window titlebar (see picture).

[Pic 1](http://realityisreality.com/mate-marco-compton.png)

If I use Compiz, there is no line through the window titlebar, but the text in the title bar is not scaled (i.e. it is tiny) (see picture).

[Pic 2](http://realityisreality.com/mate-compiz.png)

Any advice/workaround much appreciated!

**Comment by raveit65** [comment]
I can confirm that with traditional themes.
I am sure it is a problem with marco because metacity WM doesn't have this problem with TOK theme.

**Comment by markd89** [comment]
Thanks, @raveit65!

I'm not familiar with TOK theme and my quick search didn't turn anything up. Can you post a link?

Thanks again,
Mark

**Comment by vkareh** [comment]
This seems like an oversight on my part :stuck_out_tongue_closed_eyes:

**Comment by raveit65** [comment]
> I'm not familiar with TOK theme and my quick search didn't turn anything up. Can you post a link?

TraditionalOK (TOK) is from mate-themes package https://github.com/mate-desktop/mate-themes/tree/master/desktop-themes/TraditionalOk
But i thought you're using it in your screenshot :)
As workaround you can use another metacity-theme from us which don't have this issue.
EG. Menta, Menta-Blue, BlackMATE, Blue or Green-Submarine.
Note, the metacity-theme can be chosen independent from gtk+ theme

**Comment by raveit65** [comment]
@vkareh 
I think it is a problem with handling a color gradient.

**Comment by markd89** [comment]
I tried the other themes and the line through title bar is fixed... BUT,  I prefer the TOK themes mostly because of the familiar minimize, restore, close window buttons. I'm staying subscribed here in hopes there will be a fix.

Thanks much for the fast response and the cool project, of course :-)

**Comment by markd89** [comment]
Hi Guys, so I switched back to 1920x1080. I had a couple of other things that were difficult in HiDPI and to paraphrase Bill Gates, 1920x1080 should be enough pixels for anyone.

I did notice one other HiDPI issue and wanted to pass it along: I have a system monitor graph placed in the top right panel. The graph should be vertically level with the bottom of that top panel. After a reboot, however the graph is half off the screen. 

If I right click, preferences and click OK, it would resize normally.
![misshaped-cpu](https://user-images.githubusercontent.com/690446/49060146-13532b80-f1c1-11e8-92c8-b3d0fae27295.png)

Here are the settings:

![cpugraph settings](https://user-images.githubusercontent.com/690446/49060153-18b07600-f1c1-11e8-8f91-2eac8a20db5d.png)
